css-fonts-5/Overview.bs

```diff
@@ -59,7 +60,78 @@ rules defined in CSS Fonts Level 4.
 This specification is currently a delta to the CSS Fonts Level 4 specification.
 Do not assume that if something is not here, it has been dropped.
 
+<h2 id="text-scale-meta">
+Text-Scale <code class=html>&lt;meta&gt;</code> element</h2>
```

Comment:
I noticed the HTML spec also mentions when a metadata name should not appear more than once in the document. Maybe you should add this line somewhere:

> There must not be more than one [meta](https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element) element with its [name](https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-name) attribute value set to an [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive) match for [text-scale](https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme) per document.

Reply:
Done